### ISSUE TITLE
Add proxigram for Instagram

### ISSUE THREAD

**Issue by Troughy** [mendel5/alternative-front-ends#235]
[Proxigram](https://codeberg.org/proxigram/proxigram) is a good front-end for Instagram. I don't know how popular Bibliogram used to be, but I feel like Proxigram is underrated, even though it's the only alternative frontend for IG right now (correct me if I'm wrong). You can use it without JS, over TOR, without any issues. However there's only 1 public instance which works (until [mine](https://pg.zorby.top) gets added)

**Comment by olehkrupko** [comment]
Proxigram seems to not be working now

**Comment by Troughy** [comment]
> Proxigram seems to not be working now

Yeah well seems like the dev just forgot about it, I also shut down my instance because it just doesn't work anymore.
